pkg/ovs/openflow/interfaces.go

```diff
@@ -137,8 +150,9 @@ type OFEntry interface {
 type Flow interface {
 	OFEntry
 	MatchString() string
-	// CopyToBuilder returns a new FlowBuilder that copies the matches of the Flow, but does not copy the actions.
-	CopyToBuilder() FlowBuilder
+	// CopyToBuilder returns a new FlowBuilder that copies the matches of the Flow, but does not copy the actions. Reset
+	// the priority in the new FlowBuilder if the provided value is not 0.
```

Comment:
It resets the priority of the new FlowBuilder...
Or: The priority of the new FlowBuilder is reset...

Reply:
Done.